src/app/modules/item/components/add-dependency/add-dependency.component.html

```diff
@@ -0,0 +1,11 @@
+<alg-section icon="pi pi-plus" i18n-label label="Create a new dependency">
+  <alg-add-content
+    [allowedTypesForNewContent]="allowedNewItemTypes"
+    [searchFunction]="searchFunction"
+    [addedIds]="addedIds"
+    [loading]="false"
+    [showCreateUI]="false"
+    (contentAdded)="addDependency($event)"
+    inputCreatePlaceholder="Enter a title to create a new dependency"
```

Comment:
I think this label `inputCreatePlaceholder` should be translated, right?

Reply:
Fixed. Also I've fixed it for `add-group.component`